src/components/Axis/Ticks.tsx

```diff
@@ -33,14 +33,14 @@ export interface TicksProps extends Omit<TickProps, 'line' | 'text'> {
 
 export function Ticks(props: Omit<TicksProps, 'children'>) {
   const {
-    show,
+    hidden = false,
```

Comment:
```suggestion
    hidden,
```

The prop is mandatory